### PR TITLE
docs

### PR DESCRIPTION
**Pull request by echatav**
This is the same as the previous PR #59 but on `dev` instead of `master`

### CODE REVIEW

**Comment by echatav** [comment]
@KentShikama can you approve this one as well? Thanks

File: README.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
-[![Build Status](https://travis-ci.org/rchain/rchain.svg?branch=dev)](https://travis-ci.org/rchain/rchain)
```
Comment (KentShikama) [inline README.md]: I like this sticker :)

**Comment by echatav** [comment]
@KentShikama sorry, I added the travis status back, can you re-approve?

**Comment by KentShikama** [comment]
LGTM

On November 29, 2017 10:51:40 AM PST, Eitan Chatav <notifications@github.com> wrote:
>@KentShikama sorry, I added the travis status back, can you re-approve?
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/rchain/rchain/pull/67#issuecomment-347958629

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by echatav** [comment]
@KentShikama It's possible I just need to request merge permissions. Are you forced to get a review (as in the Github review process) to merge PRs? I am unable to merge without another review.

**Comment by pyrocto** [comment]
Kent, are you actually going to the "Files changed" tab, clicking the green "Review changes" button, selecting the "Approve" radio button, and clicking "Submit review"?  LGTM isn't sufficient to enable Eitan to submit.  (I'd do it, but I want to make sure that your review will work.)

**Comment by KentShikama** [comment]
I thought I approved this earlier. On mobile now so I can't approve. I'll do so when I get back. 

On November 29, 2017 11:17:39 AM PST, Mike Stay <notifications@github.com> wrote:
>Kent, are you actually going to the "Files changed" tab, clicking the
>green "Review changes" button, selecting the "Approve" radio button,
>and clicking "Submit review"?  LGTM isn't sufficient to enable Eitan to
>submit.  (I'd do it, but I want to make sure that your review will
>work.)
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/rchain/rchain/pull/67#issuecomment-347966358

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by echatav** [comment]
Ah, yes, you did approve it earlier, but then I made a change (the travis status which you noted I had accidentally nixed) and unfortunately that requires another approval.

**Pull request merged**